packages/features/workspace_screen/lib/src/states/workspace_state_notifier.dart

```diff
@@ -20,10 +35,7 @@ class WorkspaceStateNotifier extends StateNotifier<WorkspaceState> {
     return result;
   }
 
-  void updateLastSavedCommandCount() => state = state.copyWith(
-        updatedLastSavedCommandCount: _commandManager.count,
-      );
-
+  // Toggle the fullscreen state
```

Comment:
comment should not be needed as the function name is descriptive enough

Reply:
Everything else looks good, please delete the comment then I can merge

Reply:
@juliajulie95 I have removed the comment